### PR TITLE
Update NOTES.txt

### PR DESCRIPTION
**Pull request by sdodsley**
Make NOTES support OpenShift and tidy up the file

### CODE REVIEW

File: pso-explorer/templates/NOTES.txt [hunk 1]
```diff
@@ -17,10 +17,14 @@
 (_______/|/     \||/       (_______/(_______)|/   \__/(_______/|/   \__/
 
 
-Get the IP address of PSO Explorer by running these commands in the
-same shell:
-
-{{- if contains "NodePort" .Values.service.type }}
+{{- if .Capabilities.APIVersions.Has "route.openshift.io/v1" }}
+  export ROUTE=$(kubectl get route --namespace {{ .Release.Namespace }} | awk 'FNR>1 {print $2}' -)
```
Comment (dnix101) [inline pso-explorer/templates/NOTES.txt:21]: Shouldn't this be `oc get route`??

**Review by dnix101 — CHANGES_REQUESTED** [review]
Can we use `kubectl` for OCP? or should we use `oc`? Alternatively, we might want to use the lookup function.

File: pso-explorer/templates/NOTES.txt (hunk 1 above)
Comment (sdodsley) [reply pso-explorer/templates/NOTES.txt]: Yes - Done

**Review by dnix101 — APPROVED** [review]
LGTM

**Pull request merged**